Policies/Subscription/Azure_Subscription_SI_Dont_Use_B2C_Tenant/Security - Subscription - DontUseB2CTenant.json

```diff
@@ -0,0 +1,27 @@
+{
+    "mode": "All",
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "The effect determines what happens when the policy rule is evaluated to match"
+        },
+        "allowedValues": [
+          "Audit",
+          "Disabled",
+          "Deny"
+        ],
+        "defaultValue": "Audit"
+      }
+    },
+    "policyRule": {
```

Comment:
As this is partial let's not push this policy.

Reply:
reverting back